### PR TITLE
Adding functionality to YourOutfit.jsx component

### PR DESCRIPTION
**Pull request by cwg83**

### CODE REVIEW

File: client/src/components/Related/YourOutfit.jsx [hunk 1]
```diff
@@ -1,43 +1,61 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
```
Comment (cwg83) [inline client/src/components/Related/YourOutfit.jsx:1]: @sm-fm here is the functionality I added to YourOutfit - the commits before this are for my previous PR

File: client/src/components/App.jsx [hunk 2]
```diff
@@ -94,11 +88,11 @@ const App = () => {
...
-        stars.push(fullStar(i, 'e', key))
+        stars.push(fullStar(i, 'e', key));
       }
     }
     return stars;
-  }
+  };
 
   return (
     <>
```
Comment (sm-fm) [inline client/src/components/App.jsx:98]: This all looks good. We might want to consider moving these functions to their own file to make App.jsx look cleaner and be easier to read

File: client/src/components/Related/ProductCard.jsx [hunk 3]
```diff
@@ -1,42 +1,62 @@
...
+  }, []);
+
+  const routeChange = () => {
+    const path = `/${props.product.id.toString()}`;
+    navigate(path);
+  };
+
+  const displayComparison = () => {
+    return;
+  };
```
Comment (sm-fm) [inline client/src/components/Related/ProductCard.jsx:25]: Not sure if you're planning on doing something with this later, but if not, I would delete this.

File: client/src/components/Related/Related.jsx [hunk 4]
```diff
@@ -8,9 +8,9 @@ const Related = (props) => {
   const [relatedProducts, setRelatedProducts] = useState();
   const [isFetching, setIsFetching] = useState(true);
 
-  const handleSetIsFetching = () => {
-    setIsFetching(!isFetching);
-  };
+  // const handleSetIsFetching = () => {
+  //   setIsFetching(!isFetching);
+  // };
```
Comment (sm-fm) [inline client/src/components/Related/Related.jsx:13]: Friendly reminder to delete these comments

**Review by sm-fm — COMMENTED** [review]
Looks good!
Had a few comments over specific lines mostly related to deleting comments/unused code.

**Pull request merged**